src/Features/Core/Portable/SolutionCrawler/IDocumentTrackingService.cs

```diff
@@ -1,32 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
```

Comment:
this was moved to Workspace level, but was cleaned up so much, it shows as a remove+add.